From 9103bea7aa8db70d6d01a75e8295b5b8d8c7d287 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:43:37 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(c56e3c2)..lineage-16.0-android-9.0.0_r31(7a87aa6) (DO NOT
 SUBMIT)

Bill Yi (1):
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5115288 from 91ce6a8edf213777a1037cd5b720977ca95faec0 to
   pi-qpr2-release

Change-Id: I7e2f4a537e372551728299e1ce4ee5ad53741e3f

Change-Id: Ib341fa2da15033ffc56e1d5c5ab47d4020b6f6fd
---
 res/values-ar/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-ar/strings.xml b/res/values-ar/strings.xml
index 321ec00..7ce4b1b 100644
--- a/res/values-ar/strings.xml
+++ b/res/values-ar/strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_label" msgid="7454398782183407432">"سعة تخزين التلفزيون"</string>
+    <string name="app_label" msgid="7454398782183407432">"مساحة تخزين التلفزيون"</string>
     <string name="permlab_readTvListings" msgid="5272001606068276291">"قراءة جميع قوائم التلفزيون"</string>
     <string name="permdesc_readTvListings" msgid="1165157606059567535">"قراءة جميع قوائم التلفزيون المتاحة على جهازك"</string>
 </resources>
-- 
2.17.1

